tests/tensorflow_autolog/test_tensorflow_autolog.py

```diff
@@ -8,7 +8,7 @@
 import numpy as np
 import pandas as pd
 import tensorflow as tf
-from tensorflow.python.keras import layers
+from tensorflow.python.keras import layers  # pylint: disable=import-error
```

Comment:
pylint checks if modules are importable or not. I'm wondering if this is really necessary because:

1. We can detect unimportable pacakges when we run test scripts.
2. We have to install all packages to suppress pylint errors. This drastically increases the time to run lint checks which should be done quick.

Reply:
It's out of the scope of this PR though.

Reply:
Agree that this check seems unnecessary

Reply:
I'll address this in a follow-up.